src/classes/issues-processor.ts

```diff
@@ -76,6 +76,8 @@ export class IssuesProcessor {
   readonly deletedBranchIssues: Issue[] = [];
   readonly removedLabelIssues: Issue[] = [];
   readonly addedLabelIssues: Issue[] = [];
+  readonly addedStaleCommentIssues: Issue[] = [];
```

Comment:
are these being added just for the test?

Reply:
maybe we could just inspect the `statistics._addedIssuesCommentsCount` so we don't have to add a field just for tests?

Reply:
Actually, it's consistency regarding all this kind of hacks that are indeed all used to only facilitate the tests.
But yeah, using the stats would be possible.